### DISCUSSION TITLE
Does anybody have this Novatation Super Bass Station panel?

### DISCUSSION THREAD

**Discussion by thx1200** [RomanKubiak/ctrlr#539] [General]
I can't sign into the forums any more and I can't register a new account.  Does anybody have a copy of this (or any other) Novation Super Bass Station panel?  https://ctrlr.org/forums/topic/novation-super-bass-station-panel/

**Comment by damiensellier** [comment]
The forum is locked and impossible to download attached files unfortunately.
The author has another panel for a kawai synth and on this panel, the panel author field is going to  https://soundcloud.com/thrax-1
You can probably contact him via soundcloud he will send it to you. This is my investigation as far as I can go ;)
Hope it helps
Damien

**Comment by thx1200** [comment]
Thanks dude!  I'll send him a message.  :-)

**Reply by subv0x** [reply]
wow it's hard to log on there now; millions of captcha demands, and then having to log on to wordpress? 
you can still access a lot of panels in the downloads.

**Reply by thx1200** [reply]
I was never able to sign in or make contact with the original author.  Were you able to?  I am unable to download that panel but still would like to.

**Reply by subv0x** [reply]
i'll have a look on an external drive to see if i grabbed it. it's a shame no attachments are accessible. another resource swallowed by the internet's amnesia.

**Reply by damiensellier** [reply]
I just wrote to the dev via soundcloud for you ;) I will let keep you posted!

**Reply by thx1200** [reply]
Nice!  Yes I wrote to him too, no reply.  Maybe if more people show interest, he will pop up.  :-)
